src/spatial/models/heatmapModel.py

```
@@ -394,13 +394,20 @@ def process_data(self) -> bool:
 
 
 # ----------------------------- AirQualityGrids ---------------------------- #
+import pandas as pd
```

Ruff (0.12.2)

Redefinition of unused `pd` from line 9

Remove definition: `pd`

(F811)